packages/components/src/components/TrezorLogo/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+const Img = styled.img``;
+
+interface Props extends React.ImgHTMLAttributes<HTMLImageElement> {
+    type: string;
+}
+
+const TrezorLogo = ({ type, ...rest }: Props) => {
+    // $FlowIssue: `require` must be a string literal.
```

Comment:
should be deleted since we are not using flowtype

Reply:
deleted